### PR TITLE
[SCHEMA] Define YAML tables for PET common metadata fields and radiopharmaceutical data v2

### PR DESCRIPTION
**Pull request by bendhouseart**
References #1014 and #1002 

Aped from tsalo's #1017, but just PET fields.

Changes proposed:

Add YAML definitions for PET page's common metadata sections.

Fixes ugly history view from previous (cancelled) PR #1026

### CODE REVIEW

File: src/schema/rules/sidecars.yml [hunk 1]
```diff
@@ -6,6 +6,189 @@
...
+            level: RECOMMENDED
+        MolarActivity:
+            level: RECOMMENDED
+        MolarActivityUnits:
+            level: RECOMMENDED
+        MolarActivityMeasTime:
+            level: RECOMMENDED
+        InfusionRadioactivity:
+            level: RECOMMENDED
+            addendum: REQUIRED if PETRadioChemistry.fields.ModeOfAdministration == 'bolus-infusion'
```
Comment (tsalo) [inline src/schema/rules/sidecars.yml:82]:
```suggestion
            level_addendum: REQUIRED if PETRadioChemistry.fields.ModeOfAdministration == 'bolus-infusion'
```

At least that's what I have in #1017.

Comment (bendhouseart) [reply src/schema/rules/sidecars.yml:82]: Ah, then in that case I need to refactor every `addendum` to `level_addendum` then don't I? Will do so on next commit, after this suggestion.

Comment (tsalo) [reply src/schema/rules/sidecars.yml]: Just the ones that should appear in the "Requirement Level" column. The ones that should appear in the "Description" column should still have an "addendum".

Comment (bendhouseart) [reply src/schema/rules/sidecars.yml]: Ahh whoops.

**Comment by tsalo** [comment]
I saw that here as well: https://github.com/bids-standard/bids-specification/pull/1017/files#r814328092

I ended up dropping the quotes but @effigies proposed (https://github.com/bids-standard/bids-specification/pull/1017/files#r824140193) `dataset.modalities contains "PET"`

File: src/schema/rules/sidecars.yml [hunk 2]
```diff
@@ -61,21 +246,21 @@ MRISequenceSpecifics:
 
 PETMRISequenceSpecifics:
     selectors:
-        - modality == "MRI"
-        - "PET" in dataset.modalities
+    - modality == "MRI"
+    - PET in dataset.modalities
```
Comment (bendhouseart) [inline src/schema/rules/sidecars.yml:250]:
```suggestion
    - dataset.modalities contains "PET"
```

**Comment by bendhouseart** [comment]
Switched PET in to -> `dataset.modalities contains "PET"` per your suggestion. Went right over my head if you mentioned this yesterday.

**Comment by tsalo** [comment]
@bendhouseart can you move your proposed changes into a new `src/schema/rules/sidecars/pet.yaml` file to deal with the conflict?

**Comment by bendhouseart** [comment]
@tsalo done per your request.

**Comment by effigies** [comment]
Squashed and merged manually to ensure minimal destruction.

Compare [e332798](https://github.com/bids-standard/bids-specification/commit/e332798a497eaa144629e6098971d4d09c1f5e45) to validate.

**Pull request closed without merging**